README.md

```diff
@@ -17,8 +17,14 @@ in user interface (UI) environments.
 
 ### Requirements
 
-To build the binary font files from source, you need to have installed the
-[Adobe Font Development Kit for OpenType](https://github.com/adobe-type-tools/afdko/) (AFDKO).
+To build the binary font files from source, you need Python 3 and with the
+[Adobe Font Development Kit for OpenType](https://github.com/adobe-type-tools/afdko/) (AFDKO) and
+[FontTools](https://github.com/fonttools/fonttools) packages, which you can install with
+
+```sh
+sudo apt install python3 python3-pip python3-dev
```

Comment:
Remove this install command since it's specific to Linux. Whomever is building from source can find out —thru other means— what they need to do to install Python 3 on their platform.